includes/init_includes/init_ipn_postcfg.php

```diff
@@ -19,6 +19,6 @@
  */
 if (!isset($_SESSION['language'])) $_SESSION['language'] = 'english';
 
-require(zen_get_file_directory(DIR_FS_CATALOG . DIR_WS_LANGUAGES . $_SESSION['language'], 'checkout_process.php', 'false'));
+require(zen_get_file_directory(DIR_FS_CATALOG . DIR_WS_LANGUAGES . $_SESSION['language'], '/checkout_process.php', 'false'));

```

Comment:
The slash really should be after `$_SESSION['language']` instead of before `checkout_process.php` unless the use of a template override is purposefully necessary.

All other properly formatted and operational instances of `zen_get_file_directory` present the folder path with a slash at the end rather than before the filename and the function when so presented supports a template override.